packages/fluentui/react-northstar/src/components/Form/FormButton.tsx

```diff
@@ -16,15 +16,13 @@ export const formButtonClassName = 'ui-form__button';
 /**
  * An FormButton renders a Button wrapped by FormField.
  */
-export const FormButton = compose<'div', FormButtonProps, FormButtonStylesProps, SelectedFormFieldCustomProps, {}>(
-  _FormFieldBase,
+export const FormButton = compose<'button', FormButtonProps, FormButtonStylesProps, SelectedFormFieldCustomProps, {}>(
```

Comment:
Might not be so easy since the styles of the component are determined by component display name if I understand N* styling correctly, you'll need to figure out a way to retain the button styles here.